CodePush.m

```diff
@@ -100,16 +103,12 @@ + (void)setUsingTestConfiguration:(BOOL)shouldUseTestConfiguration
 }
 
 /*
- * This is used to clean up all test updates. It can only be used
- * when the testConfigurationFlag is set to YES, otherwise it will
- * simply no-op.
+ * WARNING: This cleans up all downloaded and pending updates.
  */
-+ (void)clearTestUpdates
++ (void)clearUpdates
```

Comment:
Should we also clear out failed updates? The Cordova plugin does this and I think it makes sense since those updates may work not that there is a new binary.